### ISSUE TITLE
1.17 fabric server not working

### ISSUE THREAD

**Issue by Gabocota** [Earthcomputer/EnchantmentCracker#266]
![image](https://user-images.githubusercontent.com/88735758/128906095-28d1d21c-d1ac-408f-b23a-1c8c75541470.png)

This is my set up that worked in 1.16.5. the piston on the top covers the beacon and i am not killing any of the pigmen. 

I get my player seed correctly but when i follow the instructions, drop the items and enchant the dummy. The "wanted item" comes out with mos of the enchantment missing. Example: 
![image](https://user-images.githubusercontent.com/88735758/128906461-3d8167b8-3268-45c7-8d2d-dd6667cbb2b5.png)
![image](https://user-images.githubusercontent.com/88735758/128906495-10b78936-c434-4ce6-b030-42281bc07eee.png)

Output: 
![image](https://user-images.githubusercontent.com/88735758/128906533-197f4d52-7e23-40e0-8731-32d7e98ddb03.png)

Please help. I need a whole shulker full of god bows.

**Comment by Earthcomputer** [comment]
What mods do you have installed?

**Comment by Gabocota** [comment]
> What mods do you have installed?

Client:
![image](https://user-images.githubusercontent.com/88735758/128907259-01e3a13c-c64d-4bb0-904e-e1f418852096.png)

Server: 
![image](https://user-images.githubusercontent.com/88735758/128907326-b6228766-3661-4dd9-8466-f1f1fc27cc76.png)

**Comment by Gabocota** [comment]
Well I guess villagers aren't that bad after all

**Comment by Earthcomputer** [comment]
It's probably one of the mods you have installed messing it up, or else you're doing something wrong

**Comment by ImUrX** [comment]
You cant get xp orbs while trying to crack the seed, maybe its that?

**Comment by Earthcomputer** [comment]
Yes that might be it, you can no longer collect XP while doing enchantment cracking in 1.17

**Comment by Gabocota** [comment]
I know I can't get XP while doing it. I turn off the farm before doing it obviously.
